src/js/components/Banner.js

```diff
@@ -9,25 +9,29 @@ export default class Banner extends React.PureComponent {
     title: '',
     message: '',
     toggleBannerVisibilty: false,
-    year: 0,
-    day: 0,
-    month: 0,
+    date: 0,
+    mysqlDate: 0,
+    index: 0,
   };
 
   componentDidMount() {
     const d = new Date();
-    const $yr = d.getFullYear();
-    const $month = d.getMonth();
-    const $day = d.getDate();
-    this.setState({ year: $yr, day: $day, month: $month })
-    fetch(`http://api.sikhitothemax.org/messages/web/${$yr}-${$month}-${$day}`)
+    const $mysqlDate = d.getFullYear() + '-' + (d.getMonth() + 1) + '-' + d.getDate() + ' ' + d.getHours() + ':' + d.getMinutes() + ":" + d.getSeconds();
```

Comment:
Can you make use of `dateMath` functions we have at line 388 of `util/index.js` for this?

Reply:
Also, can you explain a bit what we are trying to do here with two states: mysqlDate and date, I think we can come up with a bit simpler logic to do that.

Reply:
Mysql date to compare to expiration date of msg and date for saving to local storage

Reply:
dateFunctions doesnt seem to have what im looking for